server/mobs.cpp

```diff
@@ -291,6 +297,72 @@ Mob::~Mob()
 	}
 }
 
+struct MultiplayerMechanicsInfo {
+	uint16_t multikill = 0;
+	uint16_t killing_spree = 0;
+	uint16_t shutdown = 0;
+	uint16_t domination = 0;
+	uint16_t revenge = 0;
+	uint16_t comeback = 0;
+};
+
+MultiplayerMechanicsInfo handleMultiplayerMechanics(Player& killer, Player& victim) {
```

Comment:
"handle multiplayer mechanics" could mean anything in a multiplayer game, maybe we could rename this to something more descriptive like for instance `handleKillstreakMechanics` or `handleKillStatistics`